# Patch settings app restricting ringtones to system ringtone app
# Partially revert Change-Id: Ie19cbceb89842a75a180898abcce81b63c18d46c
# See https://gitlab.com/LineageOS/issues/android/-/issues/9199
--- a/src/com/android/settings/RingtonePreference.java
+++ b/src/com/android/settings/RingtonePreference.java
@@ -74,8 +74,7 @@ public class RingtonePreference extends Preference {
                 true);
         mShowSilent = a.getBoolean(com.android.internal.R.styleable.RingtonePreference_showSilent,
                 true);
-        setIntent(new Intent(RingtoneManager.ACTION_RINGTONE_PICKER)
-                .setPackage(context.getString(R.string.config_sound_picker_package_name)));
+		setIntent(new Intent(RingtoneManager.ACTION_RINGTONE_PICKER));
         setUserId(UserHandle.myUserId());
         a.recycle();
     }
